test/cases/api.sh

```diff
@@ -194,12 +194,10 @@ function cleanupAWS() {
   AWS_SNAPSHOT_ID="${AWS_SNAPSHOT_ID:-}"
 
   if [ -n "$AWS_CMD" ]; then
-    set +e
```

Comment:
This will do the opposite of what you want I think? `set -e` means fast-failing.

Reply:
I've removed it from the individual functions and put it to the function calling these functions.

Reply:
Yup only noticed after, looks good :)